languages.json

```diff
@@ -368,6 +375,12 @@
       "quotes": [["\\\"", "\\\""], ["'", "'"]],
       "extensions": ["ebuild", "eclass"]
     },
+    "EdgeQL": {
+      "name": "EdgeQL",
+      "line_comment": ["#"],
+      "quotes": [["'", "'"], ["\\\"", "\\\""], ["$", "$"]],
+      "extensions": ["esdl"]
+    },
```

Comment:
This should be two different languages, `EdgeQL` and `EdgeQL Schema` schema files are `esdl` and EdgeQL query files are `edgeql`.

Reply:
Sorry for that mistake, I haven't used EdgeDB before. I've fixed it now.